### DISCUSSION TITLE
dynamic styling and filtering

### DISCUSSION THREAD

**Discussion by jaigsingla** [pelicanmapping/osgearth#2240] [Support Q&A]
Hi All,

Want to visualize the data in dynamic styling and filtering .

For example , based on country population , circle size and different color shades shall be visualized ..

or based on population , vertical line size strokes shall be identified .

thanks,
Jai

**Comment by gwaldron** [comment]
osgEarth has nothing built in for this. You will need to write custom code.

**Reply by jaigsingla** [reply]
Can we do Distinct styling for selected objects?? 
Pls guide if possible
